### ISSUE TITLE
Flow Text-UI inconsistency with the CL OS installer between the main and advanced screens

### ISSUE THREAD

**Issue by marioroy** [clearlinux/distribution#2794]
Text-mode UI Installation

The flow when making selections works wonderfully on the main screen, taking you to the next item. Unfortunately, the advanced screen takes you to the top of the screen after leaving a sub-screen, requiring scrolling from the top.

For example, assign hostname, add kernel argument, select lts kernel, and disable software updates. After exiting the sub-screen, it takes you back to the top of the screen versus the next item.

In summary, the flow is awkward on the advanced screen requiring repeat scrolling from the top.

**Comment by bryteise** [comment]
Ah yea, last time I used the installer I had similar experiences and assumed I was just doing something wrong with my navigation choices. I'll see about poking this sometime.
